server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -271,6 +271,15 @@ public class TestSqlHostVariable {
                   + "       04 {$*VAR`->VAR`->VAR-LENGTH`->VAR-DATA} USAGE IS SQL TYPE IS XML AS CLOB (10) OCCURS {123456|1} TIMES.\n"
                   + "        PROCEDURE DIVISION.\n";
 
+  public static final String LOB_XML_ARR_TEXT3_ERR2 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR-NAME1}.\n"
+                  + "       04 {$*VAR`->VAR`->VAR-LENGTH`->VAR-DATA} USAGE IS SQL TYPE IS XML AS CLOB-FILE OCCURS {123456|1} TIMES.\n"
```

Comment:
I can build the example without errors.
Also the transformation is a bit more complicated:
```
       04  VAR OCCURS 123456.                                           
         49 VAR-NAME-LENGTH PIC S9(9) COMP-5 SYNC.                      
         49 VAR-DATA-LENGTH PIC S9(9) COMP-5.                           
         49 VAR-FILE-OPTION PIC S9(9) COMP-5.                           
         49 VAR-NAME PIC X(255) SYNC.                                   
      *04 VAR USAGE IS SQL TYPE IS XML AS CLOB-FILE OCCURS 123456 TIMES.
```